### PR TITLE
FIX: allow setting custom http scheme

### PR DESCRIPTION
**Pull request by IgorNadj**
Allows you to use http on your local rather than having to use https. E.g. set local env variable `API_HOST=http://api-app`

### CODE REVIEW

File: src/Router.php [hunk 1]
```diff
@@ -13,26 +13,17 @@
 class Router
 {
 	/**
-	 * Returns full URL for a requet route
+	 * Returns full URL for a request route
```
Comment (andrewmclagan) [inline src/Router.php:16]: ha!

File: src/Router.php [hunk 2]
```diff
@@ -13,26 +13,17 @@
...
 	{
-		switch (App::environment()) {
-			case 'testing':
-				$scheme = 'http://';
-				break;			
-			default:
-				$scheme = 'https://';
-				break;
-		}
+		$host = env('API_HOST') ?? 'https://api.ethicaljobs.com.au';
```
Comment (andrewmclagan) [inline src/Router.php:23]: Much better... In future I'd even like the default to be `api-app`

**Review by andrewmclagan — APPROVED** [review]
This looks great. We should discuss  the implications for sem-ver when this gets picked up in a release. Its essentially a breaking change as its different from the behaviour previously. Although its also a FIX as the previous behaviour was invalid.

Id be happy with a bug fix version bump `2.4.X` so its picked up in all services using `2.4`. It should br fine as API_HOST will always be a fully qualified domain with scheme.

**Comment by IgorNadj** [comment]
Background reason for this change:
````
bash-4.4# curl -I https://api-app
curl: (7) Failed to connect to api-app port 443: Connection refused
````

**Pull request merged**